client/rest-high-level/src/main/java/org/opensearch/client/ClusterRequestConverters.java

```diff
@@ -132,8 +130,7 @@ static Request getComponentTemplates(GetComponentTemplatesRequest getComponentTe
     }
 
     static Request componentTemplatesExist(ComponentTemplatesExistRequest componentTemplatesRequest) {
-        final String endpoint = new RequestConverters.EndpointBuilder()
-            .addPathPartAsIs("_component_template")
+        final String endpoint = new RequestConverters.EndpointBuilder().addPathPartAsIs("_component_template")
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Similar code fragments were detected in the same file at the following lines: 133:141, 121:129.  
Refactoring can help improve code maintainability. Consider reducing duplicate code by extracting it into a separate method. You can then replace duplicated code with calls to this new method.